getting_started/access_your_cluster.adoc

```diff
@@ -0,0 +1,112 @@
+[[getting-started-accessing-your-cluster]]
+= Accessing Your Services 
```

Comment:
Shoule it be `Accessing Your Cluster`?

Reply:
I think we're still trying to come up with the proper naming of this document, since it covers more than just the `Cluster` information